zio-http/src/test/scala/zio/http/WebSocketSpec.scala

```diff
@@ -22,47 +22,47 @@ import zio.test.TestAspect.{diagnose, nonFlaky, timeout, withLiveClock}
 import zio.test.{TestClock, assertCompletes, assertTrue, assertZIO, testClock}
 
 import zio.http.ChannelEvent.UserEvent.HandshakeComplete
-import zio.http.ChannelEvent.{ChannelRead, ChannelUnregistered, UserEventTriggered}
+import zio.http.ChannelEvent.{Read, Unregistered, UserEventTriggered}
 import zio.http.internal.{DynamicServer, HttpRunnableSpec, severTestLayer}
-import zio.http.socket.{WebSocketChannelEvent, WebSocketFrame}
 
 object WebSocketSpec extends HttpRunnableSpec {
 
   private val websocketSpec = suite("WebsocketSpec")(
     test("channel events between client and server") {
       for {
-        msg <- MessageCollector.make[ChannelEvent.Event[WebSocketFrame]]
+        msg <- MessageCollector.make[WebSocketChannelEvent]
         url <- DynamicServer.wsURL
         id  <- DynamicServer.deploy {
-          Handler
-            .fromFunctionZIO[WebSocketChannelEvent] {
-              case ev @ ChannelEvent(ch, ChannelRead(frame)) => ch.writeAndFlush(frame) *> msg.add(ev.event)
-              case ev @ ChannelEvent(_, ChannelUnregistered) => msg.add(ev.event, true)
-              case ev @ ChannelEvent(_, _)                   => msg.add(ev.event)
-            }
-            .toSocketApp
-            .toRoute
+          Handler.webSocket { channel =>
+            channel.receive.flatMap {
+              case event @ Read(frame)  => channel.send(Read(frame)) *> msg.add(event)
+              case event @ Unregistered => msg.add(event, true)
+              case event                => msg.add(event)
+            }.forever
+          }.toRoute
```

Comment:
`.toHttp`?

Reply:
This is expecting an `HttpApp` that doesn't have `WebSocketChannel` as its input type anymore.